src/Command/Migrate.php

```diff
@@ -11,7 +11,7 @@
  */
 namespace Migrations\Command;
 
-use Cake\Event\EventManagerTrait;
+use Cake\Event\EventDispatcherTrait;
```

Comment:
Derp, I did not thought about that. I'll revert it. Maybe when 3.1 is released we can bump the requirement to CakePHP 3.1 ?